docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
 How to set up your system, environment, and upload file(s) into the database
 
 ## Setting up operating system
-Make sure you have an up-to-date web browser. Follow the directions [here](https://docs.microsoft.com/en-us/windows/wsl/install) to set up a bash shell, git, and python 3 (please set up all three). (add miniconda)
-* Run as administrator 
-	*  If using cmd, right-click on the program and click “Run as Administrator”
-* Skip distribution
+Make sure you have an up-to-date web browser. Follow the directions
```

Comment:
Do we need to worry about up-to-date web browsers in 2022? If we do, suggest adding more detail about this process. If not, suggest omitting this instruction.